### PR TITLE
Release 0.5.15

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,12 +18,12 @@ repository = "https://github.com/coreos/cargo-vendor-filterer"
...
-cargo_metadata = "0.11.3"
+cargo-lock = "10"
+cargo_metadata = "0.18.1"
 clap = { version = "4.1", features = ["derive", "wrap_help"] }
 flate2 = "1.0"
 hex = "0.4"
-toml = "0.5"
+toml = "0.8"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1.0.80"
```
Comment (piotr-sk) [inline Cargo.toml:28]:
```suggestion
serde_json = "1.0"
```
Cargo.lock bump indicates a change from "1.0.114" to "1.0.128", so listing .80 here does not add value in my opinion.

File: Cargo.toml [hunk 2]
```diff
@@ -18,12 +18,12 @@ repository = "https://github.com/coreos/cargo-vendor-filterer"
 [dependencies]
 anyhow = "1.0"
 camino = "1.0"
-cargo-lock = "8"
-cargo_metadata = "0.11.3"
+cargo-lock = "10"
+cargo_metadata = "0.18.1"
```
Comment (piotr-sk) [inline Cargo.toml:22]:
```suggestion
cargo_metadata = "0.18"
```
I recommend not to specify the patch version, as it is the case for anyhow or camino crates.

**Review by piotr-sk — APPROVED** [review]
LGTM. 
Nitpicking about patch versions in the Cargo.toml file.

**Comment by piotr-sk** [comment]
@cgwalters thanks for elevating my access rights. 
Should I merge this PR or do you want someone else to look at it?

**Comment by cgwalters** [comment]
> @cgwalters thanks for elevating my access rights.

Welcome! :smile: 

I went ahead and did the merge and publish: https://crates.io/crates/cargo-vendor-filterer/0.5.15

**Pull request merged**